src/main.ts

```diff
@@ -53,7 +53,9 @@ const app = createApp(App)
     loader,
     appLoginUrl: process.env.VUE_APP_LOGIN_URL as string,
     getConfig: getConfig,
-    initialise: initialise
+    getProductIdentificationPref: getProductIdentificationPref,
+    initialise: initialise,
+    setProductIdentificationPref: setProductIdentificationPref
```

Comment:
Since the key and value are the same, we can pass it this way
```suggestion
    setProductIdentificationPref,
```